android/README.md

```diff
@@ -0,0 +1,226 @@
+    2. Try clicking the "Crash" button in the app. What do you see in the RUM UI that might reflect that?
+
+## Part 3: Add http client instrumentation
+
+The Splunk Android library includes instrumentation for okhttp clients. In this part of the workshop, you'll use it when
+configuring the okhttp client that is used in the sample app.
+
+| Objective  | To add OpenTelemetry okhttp client instrumentation to the app and see client request metrics in the Splunk RUM product. |
+| ---        | ---           |
+| Duration   | 5-10 minutes  | 
+| Difficulty | Medium        |
+
+1. All the app's interactions with okhttp are found in the `com.splunk.android.workshopapp.FirstFragment` class. Open up
+   that class in Android Studio.
+2. Find the `buildOkHttpClient()` method.
+3. Change the last line of the method to read:
+   ```
+   return SplunkRum.getInstance().createRumOkHttpCallFactory(builder.build());
+   ```
+   Remember to add an import for the `com.splunk.rum.SplunkRum` class when prompted by Android Studio.
+4. Build and restart the app. Now try generating some http traffic by using the 3 buttons on the first screen of the app
+   labeled "LOGIN", "HTTP NOT FOUND", and "HTTP ERROR".
+5. Navigate to the Splunk RUM page in the Splunk Observability product.
+    1. Filter for your app and environment.
+    2. Try clicking on the http metrics and finding example sessions for the results. (These metrics are "Endpoint
+       Errors" and "Endpoint Latency")
+
+## Part 4: Add some manual instrumentation
+
+The SplunkRum class in the Android RUM library contains a bunch of options for doing manual instrumentation and for
+customizing the telemetry that is sent to Splunk RUM. In this part of the workshop, you'll try a couple of them, and you
+can explore the rest on your own.
+
+| Objective  | To add some manually instrumented spans and events to the app. |
+| ---        | ---            |
+| Duration   | 10 minutes +   | 
+| Difficulty | Advanced       |
+
+#### 1. Add a "workflow" to the app, and add some custom attributes.
+
+In Splunk's RUM implementation, a "workflow" is a custom span for which metrics will be automatically collected. In the
+Android instrumentation, these workflows are modeled as OpenTelemetry Span instances.
+
+Let's create a workflow for the user's login. This will be triggered by the button labled "LOGIN" on the first screen of
+the app.
+
+1. The code for this is in the `com.splunk.android.workshopapp.FirstFragment` class. Open up that class in Android
+   Studio.
+2. Find this block of code in the `onViewCreated` method:
+   ```
+        binding.loginButton.setOnClickListener(v -> {
+            //not really a login, but it does make an http call
+            makeCall("https://ssidhu.o11ystore.com/");
+        });
+   ```
+3. Change this block of code to look like this:
+   ```
+        Span loginSpan = SplunkRum.getInstance().startWorkflow("Login");
```

Comment:
perhaps surround this with `binding.loginButton.setOnClickListener(v -> {` otherwise it might give an impression that it is not needed

Reply:
good call. :) will change